libraries/AP_HAL_ChibiOS/hwdef/common/board.c

```diff
@@ -269,6 +269,30 @@ void __early_init(void) {
                      STM32_NOCACHE_MPU_REGION_2_SIZE |
                      MPU_RASR_ENABLE);
 #endif
+#if defined(DUAL_CORE)
```

Comment:
not critical, but prefer a fn in stm32_util.c

Reply:
done